spec/fixtures/users.yml

```diff
@@ -1,13 +1,13 @@
 DEFAULTS: &DEFAULTS
+  email: $LABEL@fixtures.org
   organization_name: Agency
-  persistence_token: <%= Authlogic::Random.hex_token %>
+  persistence_token: $LABEL_persistence_token
```

Comment:
This fixes the token duplication I mentioned above. `<%= Authlogic::Random.hex_token %>` was invoked once to generate a token, and that token was then re-used for every user record (oof). `$LABEL_persistence_token` dynamically generates a unique, human-readable token (for debugging sanity) based on the fixture label, i.e. `affiliate_admin_persistence_token`.